src/algos/simba_algo.py

```diff
@@ -30,6 +32,12 @@ def train(self):
     def eval(self):
         self.mode = "val"
         self.client_model.eval()
+    
```

Comment:
added these two methods to enable/disable logs, and enable/disable whether the output is detached (in the case of model_optimization where we need the gradients)